python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -214,7 +214,8 @@ def run_model(repo_id, test_api, in_out_pairs, local_model_hub=None, warm_up=1,
                             round(result[in_out_pair][-1][5], 2),
                             result[in_out_pair][-1][6] if any(keyword in test_api for keyword in ['int4_gpu', 'int4_fp16_gpu_win', 'int4_loadlowbit_gpu', 'int4_fp16_loadlowbit_gpu', 'fp16_gpu', 'deepspeed_optimize_model_gpu']) and not lookahead else 'N/A',
                             streaming if 'win' in test_api else 'N/A',
-                            use_fp16_torch_dtype if 'pipeline_parallel_gpu' in test_api else 'N/A'],
+                            use_fp16_torch_dtype if 'pipeline_parallel_gpu' in test_api else 'N/A',
+                            group_size],
```

Comment:
Should we make group_size value "N/A" for other test_api? Maybe confusing

Reply:
Good point, we will fix this in next PR . 😊